auto3dseg/notebooks/auto3dseg_hello_world.ipynb

```diff
@@ -20,7 +20,7 @@
   },
```

Comment:
<p>perhaps we should fix all the random factors and make this notebook deterministic?</p><p> </p><p><code>create_test_image_3d</code> could be deterministic with a hard-coded random state:</p><p>https://github.com/Project-MONAI/MONAI/blob/dev/monai/data/synthetic.py#L30</p><p> </p><pre class="ql-syntax" spellcheck="false">create_test_image_3d(..., random_state=np.random.RandomState(idx))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Added the deterministic args in the latest commit. Thanks